app/assets/javascripts/developers.js

```diff
@@ -42,14 +42,13 @@ $(function(undefined) {
 
   // DataTables
   (function() {
-    var DEVELOPER_TABLE = 'developer-table';
     var dataTables = [
-      DEVELOPER_TABLE
+      'developer-table', 'applications-table'
     ];
 
     var getAllFilters = function (table) {
       return $('*[data-datatable-filter="' + table + '"]');
-    }
+    };
```

Comment:
How dare you remove this semi-colon?

I know it was me with the hard coded strings in the first place, but I'd actually like to change this to : 

```
var dataTables = $('table.dataTable');
```

and the each loop at the bottom of the JS to read : 

```
$.each(dataTables, function (index, table) {
  var id = $(table).attr('id');
  var filters = getAllFilters(id);
  $.each(filters, dataTableEvents);
  $(table).DataTable();
});
```

...and of course remove the magic strings entirely.